scripts/bundle-image-pickup.sh

```diff
@@ -0,0 +1,272 @@
+    "\n---" >> $README_FILEPATH
+  fi
+
+  echo -e "\n#### Updated Build Version: $(date)" >> $README_FILEPATH
+  echo -e "\n| Image Name                                                             | Image Component  |\n" \
+      "|------------------------------------------------------------------------|------------------|\n" \
+      "| [postgresql-13](https://catalog.redhat.com/software/containers/rhel8/postgresql-13/5ffdbdef73a65398111b8362) | ${POSTGRES_IMAGE}  |\n" \
+      "| [search-collector](https://github.com/stolostron/search-collector)     | ${COLLECTOR_IMAGE} |\n" \
+      "| [search-indexer](https://github.com/stolostron/search-indexer)         | ${INDEXER_IMAGE}   |\n" \
+      "| [search-v2-api](https://github.com/stolostron/search-v2-api)           | ${API_IMAGE}       |\n" \
+      "| [search-v2-operator](https://github.com/stolostron/search-v2-operator) | ${OPERATOR_IMAGE}  |" >> $README_FILEPATH
+
+  sed -i'.bak' 's/^[ \t]*//'  $README_FILEPATH
+}
+
+update_images_csv () {
+  COMPONENT=$1
+  NEW_IMAGE=$2
+  IGNORE=$3
+
+  log_color purple "Preparing to update component: ${COMPONENT} => ${NEW_IMAGE}\n"
+
+  # TODO: Replace yq path with $OPERATOR_ENV_PATH. (Note: Adding the env variable seems to cause yq to return no results)
+  if [[ $COMPONENT =~ .*"postgresql-13".* ]]; then
+    yq -i e '.spec.install.spec.deployments[0].spec.template.spec.containers[1].env[1].value = "'${NEW_IMAGE}'"' $OPERATOR_CSV_FILEPATH
+  
+  elif [[ $COMPONENT =~ .*"search-indexer".* ]]; then
+    yq -i e '.spec.install.spec.deployments[0].spec.template.spec.containers[1].env[1].value = "'${NEW_IMAGE}'"' $OPERATOR_CSV_FILEPATH
+
+  elif [[ $COMPONENT =~ .*"search-collector".* ]]; then
+    yq -i e '.spec.install.spec.deployments[0].spec.template.spec.containers[1].env[2].value = "'${NEW_IMAGE}'"' $OPERATOR_CSV_FILEPATH
+  
+  elif [[ $COMPONENT =~ .*"search-v2-api".* ]]; then
+    yq -i e '.spec.install.spec.deployments[0].spec.template.spec.containers[1].env[3].value = "'${NEW_IMAGE}'"' $OPERATOR_CSV_FILEPATH
+
+  # TODO: Replace yq path with $OPERATOR_IMAGE_PATH. (Note: Adding the env variable seems to cause yq to return no results)
+  elif [[ $COMPONENT =~ .*"search-v2-operator".* && $IGNORE_POSTGRES_IMAGE_UPDATE == true ]]; then
+    yq -i e '.spec.install.spec.deployments[0].spec.template.spec.containers[1].image = "'${NEW_IMAGE}'"' $OPERATOR_CSV_FILEPATH
+  fi
+}
+
+# Fetch the current images from the csv file.
+get_images_from_csv
+
+# Set the default snapshot version.
+if [ -f ./snapshot.ver ]; then
+  DEFAULT_SNAPSHOT=`cat ./snapshot.ver`
+  DEFAULT_POSTGRESQL_SNAPSHOT=`cat ./pg-snapshot.ver`
+
+elif [[ " $@ " =~ " --silent " || " $@ " =~ " -s " ]]; then
+  log_color "red" "ERROR: Silent mode will not work when ./snapshot.ver is missing"
+  exit 1
+fi
+
+# Input or fetch the latest build image to update the csv file.
+if [[ " $@ " =~ " --silent " || " $@ " =~ " -s " ]]; then
+  log_color "cyan" "** Running in silent mode **"
```

Comment:
This is nice, please document the usage with a comment at the top of this script.